app/pluginmanager.js

```diff
@@ -346,6 +346,12 @@ PluginManager.prototype.getPackageJson = function (folder) {
 PluginManager.prototype.isEnabled = function (category, pluginName) {
   var self = this;
 
+  // MyVolumio plugins aren't very good at updating the enabled flag - just check that 
+  // the plugin is valid
+  if(self.myVolumioPluginManager.checkIfPluginIsInCurrentPlan(category, pluginName)) {
```

Comment:
You should check if `myVolumioPluginManager` is not `undefined` -- else will fail for all non myvolumio (community) build devices that don't contain myVolumio stuff.. :-)

Reply:
Yep, that's the culprit